.github/workflows/tests.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: ^1.16
+          go-version: ^1.20
```

Comment:
Don't we need to set here the same version, as in the Dockerfile? I guess that it is better to run unit tests with the same GO version than the PROD code

Reply:
Yeah. I hesitated here which version to test. I agreed to test with the same version.